### PR TITLE
fix: removed keep alive header term from dev guide

### PR DESCRIPTION
**Pull request by Arindam200**
## Description

I have removed keep alive header term from dev guide

Fixes # (issue)

## Type of change

Docs Change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue).
- [ ] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).
- [ ] This change requires a documentation update.

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

## Additional Context (Please include any Screenshots/gifs if relevant)

...

## Checklist:

- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [ ] I have made corresponding changes to the documentation.
- [ ] I have added corresponding tests.
- [ ] Any dependent changes have been merged and published in downstream modules.

### CODE REVIEW

**Review by nehagup — APPROVED** [review]

**Review by nehagup — COMMENTED** [review]
@Arindam200 can you please remove keep-alive mentions at other places in the docs as well!?

**Comment by Arindam200** [comment]
> @Arindam200 can you please remove keep-alive mentions at other places in the docs as well!?

Yes! I'm working on that!

File: versioned_docs/version-2.0.0/quickstart/node-express-mongoose.md [hunk 1]
```diff
@@ -314,18 +312,17 @@ Depending on your OS, choose your adventure:
   Your results should be looking all _snazzy_, like this:
   <img src="/docs/img/testrun-node-fail.png" alt="Sample Keploy Test Result Gin MongoDB" width="100%" style={{ borderRadius: '5px' }}/>
 
-  Our testcases failed as the Keep-Alive connection won't be available when we are using testmode, this happen because in test mode the Keploy uses the `Mocks.yml`, which was generated in the record mode. 🕰️
+  Our testcases failed because in test mode the Keploy uses the `Mocks.yml`, which was generated in the record mode. 🕰️
```
Comment (nehagup) [inline versioned_docs/version-2.0.0/quickstart/node-express-mongoose.md:315]: @Sonichigo is this statement still required?

**Review by Sonichigo — CHANGES_REQUESTED** [review]
Please modify the specified lines!

**Review by nehagup — APPROVED** [review]

**Pull request merged**